src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -321,6 +321,27 @@ final class SecureStore: Store {
 	}
 
 	#endif
+
+	// MARK: - Internal
+
+	/// Removes most key/value pairs.
+	///
+	/// Keys whose values are not removed:
+	/// * `developerSubmissionBaseURLOverride`
+	/// * `developerDistributionBaseURLOverride`
+	/// * `developerVerificationBaseURLOverride`
+	///
+	/// - Note: This is just a wrapper to the `SQLiteKeyValueStore:flush` call
+	func flush() {
+		try? kvStore.flush()
```

Comment:
I'm covering this in another branch but please don't let such things fail silently.

Reply:
Was just moving this code. Let us refactor this in another branch.